controller/api/models.py

```diff
@@ -820,12 +832,45 @@ class Domain(AuditedModel):
     owner = models.ForeignKey(settings.AUTH_USER_MODEL)
     app = models.ForeignKey('App')
     domain = models.TextField(blank=False, null=False, unique=True)
+    cert = models.ForeignKey('DomainCert', null=True)
 
     def __str__(self):
         return self.domain
 
 
 @python_2_unicode_compatible
+class DomainCert(AuditedModel):
```

Comment:
The idea behind `DomainCert` here was because certificates are intimately tied to a domain. It's cosmetic so I'd be happy to change if it would help clarify things across the board.